### PR TITLE
Epic 7 - Pastukhov Oleksandr

### PR DESCRIPTION
**Pull request by alexpastukhovnulp**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/oleksandr_pastukhov/Epic_7/control_work.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#include<iostream>
+#include<cmath>
+
+
+double** Matrix_sort(double** matrix){
```
Comment (fm2lpnu) [inline ai_12/oleksandr_pastukhov/Epic_7/control_work.cpp:5]: + Use of overloading

File: ai_12/oleksandr_pastukhov/Epic_7/control_work.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    int m = 2;
+    int c =1;
+    float x = 1.2;
+
+    double q = sqrt(m* tan(x)+abs(c*sin(x)));
+    double** arr_for_doubles = Matrix(q);
+//    int** arr_for_int = Matrix(q);
+
+    printMatrix(Matrix_sort(arr_for_doubles));
+//    Matrix_sort(arr_for_int);
```
Comment (fm2lpnu) [inline ai_12/oleksandr_pastukhov/Epic_7/control_work.cpp:75]: Task 3-6 are not implemented.

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**